ui-sdk-components-sample/app/src/main/java/com/karhoo/samples/networksdk/base/state/ViewModelContract.kt

```diff
@@ -0,0 +1,6 @@
+package com.karhoo.samples.networksdk.base.state
+
+// A contract to have predefined events to update the state of the widget.
+internal interface ViewModelContract<EVENT> {
```

Comment:
This can be removed as we can use the one in the UI SDK (they were needed for the network SDK)

Reply:
Done